facilitator/src/lib.rs

```diff
@@ -72,30 +72,34 @@ impl Write for DigestWriter {
     }
 }
 
-/// SidecarWriter wraps an std::io::Write, but also writes any buffers passed to
-/// it into a second std::io::Write.
+/// SidecarWriter wraps a vector of std::io::Writes of one type and writes all
```

Comment:
I wish, but doing that makes other methods harder to implement, because then you have to recurse through the stacked `SidecarWriters` to eventually get to the `DigestWriter` at the bottom so you can retrieve the digest over the content.